.travis.yml

```diff
@@ -1,15 +1,45 @@
-language: c
+language: python
 sudo: true
 
+services:
+  - mysql
+  - postgresql
+
 env:
   - LUA="5.1"
   - LUA="5.2"
+  - LUA="5.3"
+
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y libmysqlclient-dev libsqlite3-dev postgresql-server-dev-all
+  - sudo pip install hererocks
+  - hererocks ~/hererocks -r^ --lua $LUA
+  - export PATH=$PATH:~/hererocks/bin
+  - eval $(luarocks path --bin)
+  - luarocks install luacheck
+  - luarocks install busted
 
 install:
-  - sudo .travis/install.sh
+  - luarocks install luadbi-scm-0.rockspec
```

Comment:
I think you should use `luarocks make` command.
I think that `install` command will also download source separatly

Reply:
changed to `luarocks make`, thanks!